pygsp/graphs/nngraphs/twomoons.py

```diff
@@ -16,15 +16,15 @@ class TwoMoons(NNGraph):
         two_moons graph or a synthesized one (default is 'standard').
         'standard' : Create a two_moons graph from a based graph.
         'synthesized' : Create a synthesized two_moon
-    sigmag : float
+    kernel_width : float
```

Comment:
kernel_witht -> see my other comment

Reply:
As a derived class, it should pass `kwargs` to `NNGraph`. I'll fix that.